doc/active_docs/CMS API.json

```diff
@@ -6,8 +6,9 @@
       "operations": [
         {
           "httpMethod": "GET",
-          "summary": "List all files",
+          "summary": "File List",
```

Comment:
We usually do not touch these files directly, but change instead in https://github.com/3scale/porta/blob/master/app/controllers/admin/api/cms/files_controller.rb#L37

...and then:

```
doc:swagger:generate:cms
```

...which generates https://github.com/3scale/porta/blob/427633bf033c58076987f04d75e0b525cbe9c35f/doc/active_docs/cms.json#L9.

Some minor details might differ on the base branch you are working on (i.e. proper `docs/active_docs/CMS API.json` file now), but the overall flow to generate the docs remains the same.

Reply:
Thanks for pointing that our @guicassolato I updated the PR with the changes in the controllers to generate the equivalent documentation, using `source2swagger` in a similar way as the other controllers are documented